helm/libero-reviewer/templates/deployment-client.yaml

```diff
@@ -37,12 +37,7 @@ spec:
             - name: http
               containerPort: 9000
               protocol: TCP
-          volumeMounts:
```

Comment:
how does this container know to talk to `http://{{ include "libero-reviewer.fullname" . }}-submission:3000` now? I can see `SERVER_PORT` is passed in as an environment variable, but not that hostname

Reply:
i think thats the CLIENT_API_PROXY_URL, but it needs proper configuration and not localhost.
I'm guessing it should be http://{{ include "libero-reviewer.fullname" . }}-submission:3000